File: framework_crewai/blog_crew/config/sanity_config.py
"""
Configuração do Sanity CMS
Este arquivo contém as configurações necessárias para conexão com o Sanity
"""

import os

# Configurações do Sanity
SANITY_CONFIG = {
    "name": "crypto-frontier",
    "title": "The Crypto Frontier",
    "project_id": os.environ.get("SANITY_PROJECT_ID", "brby2yrg"),
    "dataset": "production",
    "api_version": "2023-05-03"
}

# Endpoint da API do Sanity
def get_sanity_api_url(project_id=None, dataset=None, api_version=None):
    """Retorna a URL da API do Sanity com os parâmetros configurados"""
    _project_id = project_id or SANITY_CONFIG["project_id"]
    _dataset = dataset or SANITY_CONFIG["dataset"]
    _api_version = api_version or SANITY_CONFIG["api_version"]
    
    return f"https://{_project_id}.api.sanity.io/v{_api_version}/data/mutate/{_dataset}"

File: thecryptofrontier/agentes-python/crewai-agentes-blog/src/config/sanity.config.js
/**
 * Configuração do Strapi CMS
 * Este arquivo contém as configurações necessárias para conexão com o Strapi
 */
import { defineConfig } from 'strapi'
import { visionTool } from '@strapi/vision'
import { schemaTypes } from './src/strapi/schemaTypes/index.js'

export default defineConfig({
  name: 'crypto-frontier',
  title: 'The Crypto Frontier',
  
  projectId: 'z4sx85c6',
  dataset: 'production',
  apiVersion: '2023-05-03',
  
  plugins: [
    visionTool(),
  ],

  schema: {
    types: schemaTypes,
  },
}); 